web_src/css/repo.css

```
@@ -3083,46 +3083,86 @@ tbody.commit-list {
   }
 }
 
-.pr-status {
+.ui.segment.commit-status-list {
+  padding: 0;
+  max-height: 30vh;
+  overflow-x: hidden;
+  transition: max-height 0.4s;
```

```suggestion
  transition: max-height 0.2s;
```

We generally prefer faster transitions.